pillar.example

```diff
@@ -1,9 +1,18 @@
 # vim: ft=sls
+
+common-hosts:
```

Comment:
Not super sure about having "extended YAML features" in the example?
It's a really cool method, but maybe introducing a new pillar key isn't best practise in Salt? Not sure here tbh - maybe they have something like the `.common-hosts` notation like Gitlab?

Maybe we can also have this example in an "advanced" section in the README instead or so?

Reply:
I don't think there is any `salt` way of that (there is `import_yaml` in Jinja for more complex includes but that would be overkill here). But I think with yaml it should be evident to users anyways to be able to use anchors so simple example sounds good :)